custom/icds/tasks/sms.py

```diff
@@ -51,3 +55,27 @@ def send_monthly_sms_report():
         send_html_email_async.delay(subject, recipients, message,
                                     email_from=settings.DEFAULT_FROM_EMAIL)
         raise e
+
+
+@receiver(post_save, sender=DailyOutboundSMSLimitReached)
+def send_sms_limit_exceeded_alert(sender, instance, **kwargs):
+    if settings.SERVER_ENVIRONMENT == 'icds':
+        sms_daily_limit = 10000
```

Comment:
nit: If you could put this in a constant, and update it also in `daily_limit` where it is checked for, it would make it easier to update this in future, if and when needed

Reply:
Makes sense. 
Can you please guide me on how do we handle constants in HQ?

Reply:
we keep app level constants in `const.py` file. Since this is bound to sms it should go in `corehq/apps/sms`. You would need to create a new file under that directory if it's not present already.

settings file is a good option but might be an overkill. We don't need this to change for different environments, nor its really a secret so keeping it as a const at the app level should be good.

Reply:
Done